sqlitecluster/SQLite.cpp

```diff
@@ -258,7 +258,16 @@ int SQLite::_sqliteWALCallback(void* data, sqlite3* db, const char* dbName, int
             SINFO("[checkpoint] Ready for complete checkpoint but skipping because less than 100 commits since last complete checkpoint.");
             return SQLITE_OK;
         }
-        // If we get here, then full checkpoints are enabled, and we have enough pages in the WAL file to perform one.
+
```

Comment:
NAB maybe add a comment linking to the sqlite doc for the opcode
```suggestion
// For more information on SQLITE_FCNTL_EXTERNAL_READER and usage see http://www.sqlite.org/src/artifact/18ec33e320017?ln=1132
```